task/buildah-remote/0.4/buildah-remote.yaml

```diff
@@ -656,11 +715,15 @@ spec:
       # disable host subcription manager integration
       find /usr/share/rhel/secrets -type l -exec unlink {} \;
 
+      set_proxy
+
```

Comment:
By default buildah propagates proxy variables into build, see `buildah build --http-proxy` option, which is true by default

```
       --http-proxy

       By default proxy environment variables are passed into the container if  set  for  the  Buildah  process.
       This  can  be  disabled by setting the --http-proxy option to false.  The environment variables passed in
       include http_proxy, https_proxy, ftp_proxy, no_proxy, and also the upper case versions of those.

       Defaults to true
```

However ADR says, that only layers will be cached, thus proxy shoudln't be propagated to build container

Reply:
I see your point now. I'll try setting `--http-proxy` to `false` to stop the env variable propagation.

Reply:
It worked! All the `buildah build` commands now have `--http-proxy=false` set. I verified that when defined, the proxy is used to pull image layers but not used for a `RUN` directive (see [PR #11](https://github.com/amisstea/testrepo/pull/11/files) in my test repo). The build also still works when no proxy is defined. I used the squid proxy logs to validate which traffic was passing through it.